Examples/Algorithms/TrackFitting/src/RefittingAlgorithm.cpp

```diff
@@ -71,9 +70,9 @@ ActsExamples::ProcessCode ActsExamples::RefittingAlgorithm::execute(
         ctx.geoContext, ctx.magFieldContext, ctx.calibContext,
         &track.referenceSurface(), Acts::PropagatorPlainOptions()};
 
-    const Acts::BoundTrackParameters initialParams(
+    const TrackParameters initialParams(
```

Comment:
Maybe we should just get rid of this alias? I think it'd be nicer to always refer to free or bound.